iocore/net/SSLSNIConfig.cc

```diff
@@ -130,8 +130,18 @@ SNIConfigParams::Initialize()
   sni_filename = ats_stringdup(RecConfigReadConfigPath("proxy.config.ssl.servername.filename"));
   lua_State *L = lua_open(); /* opens Lua */
   luaL_openlibs(L);
-  luaL_loadfile(L, sni_filename);
-  lua_pcall(L, 0, 0, 0);
+  if (luaL_loadfile(L, sni_filename)) {
+    Error("Loading SNI configuration - luaL_loadfile: %s", lua_tostring(L, -1));
```

Comment:
Can this be a warning??  so we can load as we did before without error-ing out if the file does not exist

Reply:
Yes, I changed it to a warning.